src/DialogMessage.ts

```diff
@@ -20,4 +21,16 @@ export namespace DialogMessage {
     export const serverRunning: string = localize('tomcatExt.serverRunning', 'This Tomcat Server is already started.');
     export const serverStopped: string = localize('tomcatExt.serverStopped', 'This Tomcat Server was stopped.');
     export const continueOnExistingServer: string = localize('tomcatExt.continueOnExistingServer', 'This Tomcat Server already exists. Do you want to continue the operation on this server?');
+
+    export function getServerPortChangeErrorMessage(serverName: string, serverPort: string): string {
+        return localize('tomcatExt.serverPortChangeError', 'Changing the server port of a running server {0} will cause errors, please change it back to {1}!', serverName, serverPort);
```

Comment:
'Changing the port of the running server {0} will cause errors. Would you like to change it back to {1}?'
And we need to have three action items 'Yes', 'More Info' and 'No'.
'More Info' leads the user to a document section that further explain the issue.